requirements.txt

```diff
@@ -12,7 +12,6 @@ Flask-RESTful==0.3.6
 itsdangerous==0.24
 Jinja2>=2.10.1
 jsonschema==2.6.0
-MarkupSafe==1.0
```

Comment:
MarkupSafe 1.0 is incompatible with later versions of jinja2 and isn't directly used by our code, so I removed it. Was added by @feng-tao